src/bundle/Resources/views/admin/role/list.html.twig

```diff
@@ -74,25 +74,25 @@
             <tbody>
             {% for role in pager.currentPageResults %}
                 <tr>
-                    <td class="ez-checkbox-cell">
+                    <td class="ez-table__cell ez-table__cell--has-checkbox">
                         {% if can_delete %}
                             {{ form_widget(form_roles_delete.roles[role.id]) }}
                         {% else %}
                             {% do form_roles_delete.roles.setRendered %}
                         {% endif %}
                     </td>
-                    <td>
+                    <td class="ez-table__cell">
                         <a href="{{ path('ezplatform.role.view', { roleId: role.id }) }}">
                             {{ role.identifier }}
                         </a>
                     </td>
-                    <td>{{ role.id }}</td>
-                    <td class="text-right">
+                    <td class="ez-table__cell">{{ role.id }}</td>
+                    <td class="ez-table__cell ez-table__cell--has-action-btns text-right">
                         {% if can_assign %}
                             <a
                                 title="{{ 'role.view.list.action.assign_to_users_or_groups'|trans|desc('Assign to Users/Groups') }}"
                                 href="{{ path('ezplatform.role_assignment.create', { roleId: role.id }) }}"
-                                class="btn btn-icon mx-3">
+                                class="btn btn-icon mx-2">
```

Comment:
I believe that maybe we should keep this as `mx-3` in order to be consistent with the offset defined in `src/bundle/Resources/views/admin/section/list.html.twig` line 97.